syntax = "proto3";

service HelloService {
    rpc HelloWorld (HelloRequest) returns (HelloResponse);
}

message HelloRequest
{
    string name = 1;
}

message HelloResponse
{
    string text = 1;
}